src/interceptors/on-request-refresh-token.ts

```diff
@@ -0,0 +1,55 @@
+import { AxiosError, InternalAxiosRequestConfig } from 'axios';
+import { decode } from 'base-64';
+import { SessionExpiryRefreshInterceptorArgs } from '../types';
+import { checkIsTokenExpired } from '../utils';
+
+global.atob = decode;
```

Comment:
I suppose this line should be moved into file with `checkIsTokenExpired` definition